packages/manifold/contracts/lazyclaim/LazyPayableClaim.sol

```diff
@@ -117,14 +122,20 @@ abstract contract LazyPayableClaim is ILazyPayableClaim, AdminControl {
         return mintBitmask & claimMintTracking != 0;
     }
 
-    function _validateMint(address creatorContractAddress, uint256 instanceId, uint48 startDate, uint48 endDate, uint32 walletMax, bytes32 merkleRoot, uint32 mintIndex, bytes32[] calldata merkleProof, address mintFor) internal {
+    function _validateTotalMax(uint32 additional, uint32 total, uint32 totalMax, bool withMax) internal pure {
+        require((totalMax == 0 || total + additional <= totalMax) && (!withMax || total + additional <= MAX_UINT_24), "Too many requested for this claim");
```

Comment:
You don't need to do a <= MAX_UINT_24 check as it will overflow.  You can verify this.